swap_meet/clothing.py

```diff
@@ -1,2 +1,26 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, fabric="Unknown", condition=0):
```

Comment:
We usually want keep our parameters in the same order of parameters used by the parent class
```suggestion
    def __init__(self, id=None, condition=0, fabric="Unknown"):
```

Reply:
This has been updated, thank you Mikelle!